solidity/test/data/sweep.ts

```diff
@@ -0,0 +1,729 @@
+      "fed3b193a5f261cbcd001a78634212d49820001e1adb3e29eb4aa11c99e7d5c77dbb" +
+      "e7803760926f57e1f9a50000000000000018182cbc30f44efa5eabbb9a2f9888a27f" +
+      "eb6d2e2f1e1461534344cf1dafd437e3a6f261cbcd001a8959a5db002000207cddca" +
+      "26ea39dd08f6345c0057300443d7720c5ab4937c2711000000000000004eb83f96a1" +
+      "f1ace06832a7eb8b3a407f04b37e211363422bf58ddeb50f20a8a54ba7f261cbcd00" +
+      "1a25011a6100000020a255594fd7ad6096e47c5b0b3a636cf0ac0dafc0dcf60277a5" +
+      "00000000000000d75ff7b7b32573d64219f81e65e61881446f68dcf47a7f5b47444b" +
+      "fd35db25f5a3a8f261cbcd001a32960f84",
+  },
+
+  chainDifficulty: 1,
+}
+
+/**
+ * `MultipleDepositsNoMainUtxo` test data represents a sweep with following properties:
+ * - 3 P2WSH and 2 P2SH deposit inputs
+ * - 1 P2WPKH sweep output
+ * - No main UTXO exists for this wallet
+ * - 6+ on-chain confirmations of the sweep transaction
+ */
+export const MultipleDepositsNoMainUtxo: SweepTestData = {
+  deposits: [
+    {
+      // https://live.blockcypher.com/btc-testnet/tx/d6a04c76aab203fe9cd8a2498bb4a8c50eb767fd95719c7790ac675ed5dec526
+      fundingTx: {
+        hash: "0x26c5ded55e67ac90779c7195fd67b70ec5a8b48b49a2d89cfe03b2aa764ca0d6",
+        version: "0x01000000",
+        inputVector:
+          "0x0101278d2c20437868e49c34bc67ace5646d20c674354e061a1683c8b55cf4" +
+          "19c20100000000ffffffff",
+        outputVector:
+          "0x0230750000000000002200200b636cbcfcde39f846bc1715a9216d53a7b882" +
+          "9d8590c54d1a1389dac31e4e125e5c3b00000000001600147ac2d9378a1c47e5" +
+          "89dfb8095ca95ed2140d2726",
+        locktime: "0x00000000",
+      },
+      reveal: {
+        fundingOutputIndex: 0,
+        depositor: "0x934B98637cA318a4D6E7CA6ffd1690b8e77df637",
+        blindingFactor: "0x4a6f267c3bfaba7c",
+        // HASH160 of 03989d253b17a6a0f41838b84ff0d20e8898f9d7b1a98f2564da4cc29dcf8581d9.
+        walletPubKeyHash: "0x8db50eb52063ea9d98b3eac91489a90f738986f6",
+        // HASH160 of 0300d6f28a2f6bf9836f57fcda5d284c9a8f849316119779f0d6090830d97763a9.
+        refundPubKeyHash: "0x28e081f285138ccbe389c1eb8985716230129f89",
+        refundLocktime: "0x1cdb1462",
+        vault: "0x0000000000000000000000000000000000000000",
+      },
+    },
+    {
+      // https://live.blockcypher.com/btc-testnet/tx/659ea860a4879acba52c1518fb44f71ed8bb0db78074da29eaf0afac6e2d79d1
+      fundingTx: {
+        hash: "0xd1792d6eacaff0ea29da7480b70dbbd81ef744fb18152ca5cb9a87a460a89e65",
+        version: "0x01000000",
+        inputVector:
+          "0x010ab93f22d004afe73c75e24a9b790854be61f8a262ff7c41022b8cce9c82" +
+          "d8730100000000ffffffff",
+        outputVector:
+          "0x02102700000000000017a914e56ff1445b513721fb0e3981817d1b17d83f95" +
+          "b087acb43a00000000001600147ac2d9378a1c47e589dfb8095ca95ed2140d27" +
+          "26",
+        locktime: "0x00000000",
+      },
+      reveal: {
+        fundingOutputIndex: 0,
```

Comment:
Would it make sense to write a testcase where `fundingOutputIndex` is grater than `0`?
In theory somebody can make a deposit without using our `makeDeposit` functionality (which always causes `fundingOutputIndex` to be `0`) and deposit UTXO could be at  an index different than `0`.

Reply:
Sure, let's add a TODO.

Reply:
See https://github.com/keep-network/tbtc-v2/pull/106/commits/b05e2a6d3a359f4973c981bcc7c605f77ef820f4